MultiThreading.Task5.Threads.SharedCollection/Program.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections.Concurrent;
```

Comment:
program doesn't work as expected. it gives me the following result:
```
Added 1 to the collection.
Added 2 to the collection.
[1, 2]
Added 3 to the collection.
[1, 2, 3]
[1, 2, 3]
Added 4 to the collection.
Added 5 to the collection.
[1, 2, 3, 4, 5]
[1, 2, 3, 4, 5]
Added 6 to the collection.
Added 7 to the collection.
[1, 2, 3, 4, 5, 6, 7]
[1, 2, 3, 4, 5, 6, 7]
Added 8 to the collection.
Added 9 to the collection.
[1, 2, 3, 4, 5, 6, 7, 8, 9]
[1, 2, 3, 4, 5, 6, 7, 8, 9]
Added 10 to the collection.
[1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
```

it should be 
```
Added 1 to the collection.
[1]
Added 2 to the collection.
[1, 2]
Added 3 to the collection.
[1, 2, 3]
Added 4 to the collection.
[1, 2, 3, 4]
Added 5 to the collection.
[1, 2, 3, 4, 5]
Added 6 to the collection.
[1, 2, 3, 4, 5, 6]
Added 7 to the collection.
[1, 2, 3, 4, 5, 6, 7]
Added 8 to the collection.
[1, 2, 3, 4, 5, 6, 7, 8]
Added 9 to the collection.
[1, 2, 3, 4, 5, 6, 7, 8, 9]
Added 10 to the collection.
[1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
```